tests/test_protocol.py

```diff
@@ -761,3 +761,197 @@ def test_records_same_packet_share_fate():
         first_time = dnsin.answers[0].created
         for answer in dnsin.answers:
             assert answer.created == first_time
+
+
```

Comment:
I wish I could come up with a better way to test this, but that is what we are testing...invalid packets